### PR TITLE
[msal-node]: Bump up the axios package

### PR DESCRIPTION
**Pull request by samuelkubai**
Bump the axios package to 0.21.1 to cover for vulnerabilities reported in #2820

### CODE REVIEW

File: change/@azure-msal-node-2021-01-05-16-31-39-bump-axios-package.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```
Comment (tnorling) [inline change/@azure-msal-node-2021-01-05-16-31-39-bump-axios-package.json:2]:
```suggestion
  "type": "prerelease",
```

File: change/@azure-msal-angular-2021-01-05-16-31-39-bump-axios-package.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
```
Comment (tnorling) [inline change/@azure-msal-angular-2021-01-05-16-31-39-bump-axios-package.json:2]:
```suggestion
  "type": "none",
```

**Review by tnorling — APPROVED** [review]

**Review by hectormmg — APPROVED** [review]

**Review by sameerag — APPROVED** [review]
lgtm

**Review by jo-arroyo — APPROVED** [review]
LGTM

**Comment by ramya-rao-a** [comment]
@tnorling, @hectormmg, @sameerag When can we expect an update to be released for `@azure/msal-node` with this change?

cc @jonathandturner

**Comment by sameerag** [comment]
@ramya-rao-a Will be releasing this early next week.

**Comment by sameerag** [comment]
@ramya-rao-a Latest node beta:  `@azure/msal-node/v/1.0.0-beta.3` is released with this change [here](https://www.npmjs.com/package/@azure/msal-node/v/1.0.0-beta.3).

**Pull request merged**